lib/ansible/module_utils/facts/hardware/darwin.py

```diff
@@ -84,13 +85,25 @@ def get_cpu_facts(self):
         return cpu_facts
 
     def get_memory_facts(self):
-        memory_facts = {}
-
-        memory_facts['memtotal_mb'] = int(self.sysctl['hw.memsize']) // 1024 // 1024
+        memory_facts = {
+            'memtotal_mb': int(self.sysctl['hw.memsize']) // 1024 // 1024
+        }
 
-        rc, out, err = self.module.run_command("sysctl hw.usermem")
+        total_used = 0
+        rc, out, err = self.module.run_command("/usr/bin/vm_stat")
```

Comment:
Use `get_bin_path()` to ensure we always can find the command in case the path changes.
```suggestion
        vm_stat_command = get_bin_path('vm_stat', True)
        rc, out, err = self.module.run_command(vm_stat_command)
```